app/models/concerns/foreman_openscap/policy_extensions.rb

```diff
@@ -143,6 +155,10 @@ def to_enc
       }.merge(period_enc)
     end
 
+    def find_assets(method)
```

Comment:
definitely a `public_send` would be better than `send` but I tend to agree with @shlomizadok, I found just two usages of this method for so I'd prefer to be explicit, something like

```ruby
def find_assets(scope)
  case scope
  when :compliant
    Scaptimony::Asset.comply_with(self)
  when :incompliant
    Scaptimony::Asset.incomply_with(self)
  else
    raise "Unsupported asset scope: #{scope}"
end
```